src/Raven.Client/ServerWide/Operations/UpdateDocumentsCompressionConfigurationOperation.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Net.Http;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Documents.Operations;
+using Raven.Client.Http;
+using Raven.Client.Json;
+using Raven.Client.Json.Serialization;
+using Raven.Client.Util;
+using Sparrow.Json;
+
+namespace Raven.Client.ServerWide.Operations
```

Comment:
Let's not put everything in one bag and create a dedicated namespace for DocumentsCompression
e.g. move it to `Raven.Client.ServerWide.Operations.DocumentsCompression`

Reply:
Done